client/src/components/ConversationPost.js

```diff
@@ -68,7 +70,28 @@ const useStyles = makeStyles(theme => ({
 
 function ConversationPost(props) {
   const classes = useStyles();
-  const { name, username, imageUrl, title, audioURL } = props;
+  const { name, username, imageUrl, title, audioURL, numLikes } = props;
+  const { userID, convoID } = props;
+
+  const handleLike = async () => {
+    console.log("liked!");
+
+    const data = {
+      userID,
+      convoID
+    };
+
+    await axios
+      .put(`/api/users/${props.id}/conversations/like`, data)
+      .then(response => {
+        console.log("Updated data ", response.data);
+      })
+      .catch(error => {
+        console.log(error);
+      });
+    window.location.reload();
```

Comment:
Will remove the reload by moving the update handler to a higher level component in the next PR